modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java

```diff
@@ -426,13 +431,16 @@ public void testGetExpectedAMImageEntriesCount() throws Exception {
 			_unregisterAMImageCounter(amImageCounterServiceRegistration);
 
 			_companyLocalService.deleteCompany(company);
+			PrincipalThreadLocal.setName(originalName);
```

Comment:
Don't we need a blank line between `_companyLocalService` and `PrincipalThreadLocal`? Same in other places.